### PR TITLE
Update project statuses and meeting info on website

### PR DESCRIPTION
**Pull request by arfazca**
**Pull Request Task List:**
- [x] Set **No Meetings Planned** for all maintained/inactive projects
- [x] Add VikeEats meeting time: Tuesdays @ 2:30pm
- [x] Mark Study Sprints as inactive
- [x] Mark CourseUp as inactive, project meeting times as "No Meetings Planned"
- [x] "Undecided" -> "No Meetings Planned"
- [x] **Inactive** buttons marked as _Red_, **Active** buttons as _Green_ and **No Meetings Planned** buttons as _Blue_

### CODE REVIEW

File: pages/projects.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const projects = [
       description: "A website built to simplify the experience of searching courses and building timetables for the University of Victoria (UVic).",
       status: "Active",
       tech: ["TypeScript", "React", "Firebase"],
-      meetingTime: "Fridays at 6-7pm",
+      meetingTime: "Waiting for updates",
```
Comment (AmandaGAnderson) [inline pages/projects.tsx]: @liamshatzel do you have an update for this?

File: pages/projects.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ const projects = [
       description: "A website for registration and management of lockers, designed with mobile-first functionality. Now maintained by the UVic ECSS.",
       status: "Maintained",
       tech: ["Svelte", "TypeScript", "PostgreSQL", "Docker"],
-      meetingTime: "Undecided",
+      meetingTime: "No Meetings Planned",
```
Comment (AmandaGAnderson) [inline pages/projects.tsx:71]: Could "No Meetings Planned" be blue? Just to make it obvious on first glance which meetings are scheduled without reading every project.

File: pages/projects.tsx (hunk 1 above)
Comment (AmandaGAnderson) [reply pages/projects.tsx]: Also, should Scholar-U have their meetings as "Undecided" or "No Meetings Planned". @liamshatzel you were talking to them right?

Comment (AmandaGAnderson) [reply pages/projects.tsx]: Confirmed in meeting, Courseup should be "Undecided" for now.

Scholar-U/Scholarship Finder should be "No Meetings Planned"

**Review by arfazca — COMMENTED** [review]
Corrected Changes as discussed in the meeting!

**Review by AmandaGAnderson — APPROVED** [review]
Looks great!

**Pull request merged**